lib/hammer_cli_foreman/subnet.rb

```diff
@@ -9,6 +9,7 @@ class ListCommand < HammerCLIForeman::ListCommand
       output do
         field :id, _("Id")
         field :name, _("Name")
+        field :description, _("Description")
```

Comment:
Unfortunately it will always show empty column. It is possible to use 
```ruby
field :description, _("Description"), Fields::Field, :hide_blank => true
```
but it's not completely the same plus it won't work in lists.

If we wanted to implement hiding unsupported fields we would probably need to rely on `foreman_version` from response headers, which is more effective than checking if the key is missing in the whole list.